tests/test_json.py

```
@@ -757,6 +762,20 @@ def test_objectid(self):
         x = json.loads(djson, cls=MontyDecoder)
         assert isinstance(x, ObjectId)
 
+    @pytest.mark.skipif(DBRef is None, reason="bson not present")
+    def test_dbref(self):
+        dbref = DBRef(
+            "my_collection", 1, database="my_database", extra_field="extra_value"
+        )
+        with pytest.raises(TypeError):
+            json.dumps(dbref)
+        djson = json.dumps(dbref, cls=MontyEncoder)
```

Ruff (0.8.2)

Local variable `djson` is assigned to but never used

Remove assignment to unused variable `djson`

(F841)